pkgs/tools/security/sequoia/default.nix

```diff
@@ -54,27 +70,24 @@ rustPlatform.buildRustPackage rec {
     "build-release"
   ];
 
-  LIBCLANG_PATH = "${llvmPackages.libclang}/lib";
+  LIBCLANG_PATH = "${llvmPackages_10.libclang}/lib";
 
-  # Please check if this is still needed when updating.
-  # Exlude tests for sequoia-store, they often error with 'Too many open files' Hydra.
-  CARGO_TEST_ARGS = " --all --exclude sequoia-store";
+  # Sometimes, tests fail on CI (ofborg) & hydra without this
+  CARGO_TEST_ARGS = "--exclude sequoia-store";
 
+  # Without this, the examples won't build
   postPatch = ''
-    # otherwise, the check fails because we delete the `.git` in the unpack phase
-    substituteInPlace openpgp-ffi/Makefile \
-      --replace 'git grep' 'grep -R'
```

Comment:
Correct. But, note the (harmless?) errors you get without it at the beginning:

```
unpacking sources
unpacking source archive /nix/store/b7h3ypagny9sz9y50kl2a0bdvywmvf3m-sequoia
source root is sequoia
unpacking source archive /nix/store/nsijfzfig03h0vdlabdw0xlkmlf708ny-sequoia-0.19.0-vendor.tar.gz
setting SOURCE_DATE_EPOCH to timestamp 315619200 of file sequoia/tool/tests/sq-sign.rs
patching sources
Validating consistency between /build/sequoia/Cargo.lock and /build/sequoia-0.19.0-vendor.tar.gz/Cargo.lock
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/j206pskg2yzyla1cnfrb9kb5n5bfvjgm-bash-4.4-p23/bin/bash PREFIX=/nix/store/42wx89vypywfm2qpvwb68zkmqgcpcxsl-sequoia-0.19.0 build-release
make -Copenpgp-ffi build-release
fatal: not a git repository (or any of the parent directories): .git
make[1]: Entering directory '/build/sequoia/openpgp-ffi'
```

Upstream seems to be relying on this in some way. I also would like to avoid the scenario where in future releases, lacking a`.git` directory will lead to errors making the build fail.

Reply:
Unless it is essential for it to I'd recommend removing `leaveDotGit`: https://github.com/NixOS/nixpkgs/issues/8567

Reply:
Upstream removes the `.git` directory from source distributions (`dist` target in the main Makefile), and they build without that error. So, I guess we should be fine with removing `.git`, too. However, I have not found out where the error line comes from :shrug: 

The signed source distributions are available at https://sequoia-pgp.org/dist/. I'd like to switch to those in the long run, and also verify the signatures :)

Reply:
> The signed source distributions are available at sequoia-pgp.org/dist. I'd like to switch to those in the long run, and also verify the signatures :)

That would be terrific indeed, but it's kind of hard to get a tarball out of these archives in a Nix expression - it seems we'd need gpg in order to verify the signature which kind of defeats the purpose of creating a new openPGP software (to get rid of gpg).

I'm switching back to `fetchFromGitLab` and hopefully we won't encounter "not a git directory" build _failures_ in the future.

Reply:
Also, the distribution tarball shouldn't be different then the gitlab tarballs, as it seems according to: https://gitlab.com/sequoia-pgp/sequoia/-/blob/v0.19.0/Makefile#L119

Reply:
That's true. Now I'm *really* confused about the `not a git repository` line. It definitely didn't show up when I ran `make -Copenpgp-ffi build-release` from the source distribution.